test/test_autocast.py

```diff
@@ -116,7 +116,7 @@ def test_autocast_torch_fp32(self):
     def test_autocast_nn_fp32(self):
         for op_with_args in self.autocast_lists.nn_fp32:
             op, args, maybe_kwargs = self.args_maybe_kwargs(op_with_args)
-            self._run_autocast_outofplace(op, args, torch.float32, module=torch._C._nn, add_kwargs=maybe_kwargs)
+            self._run_autocast_outofplace(op, args, torch.float32, module=torch.ops.aten, add_kwargs=maybe_kwargs)
```

Comment:
there's probably some python binding issue because calling into `torch._C._nn` leads to:
```
======================================================================
ERROR: test_autocast_nn_fp32 (__main__.TestAutocastCPU)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch/chourdiaanjali/work/pytorch/test/test_autocast.py", line 120, in test_autocast_nn_fp32
    self._run_autocast_outofplace(op, args, torch.float32, module=torch._C._nn, add_kwargs=maybe_kwargs)
  File "/scratch/chourdiaanjali/work/pytorch/test/test_autocast.py", line 39, in _run_autocast_outofplace
    output = getattr(module, op)(*args, **add_kwargs)
RuntimeError: SymInt constructor with input: -4771225742975334011

----------------------------------------------------------------------
Ran 1 test in 1.805s

FAILED (errors=1)
```

Reply:
ok, if you want to land this as a workaround it's good by me. I guess I'll look into the root cause then